### PR TITLE
use our own agent pool

### PR DESCRIPTION
**Pull request by alanrenmsft**
maybe we should use our own agent pool until we figure out the cause of the out of memory issue?

### CODE REVIEW

**Comment by aaomidi** [comment]
Probably unnecessary, it seems like one PR caused this?

**Comment by alanrenmsft** [comment]
> Probably unnecessary, it seems like one PR caused this?

I have explained it in my email to the team, it is not related to any PR

File: build/azure-pipelines/sql-product-build.yml [hunk 1]
```diff
@@ -7,8 +7,7 @@ resources:
 jobs:
 - job: Compile
   pool:
-    vmImage: 'Ubuntu-16.04'
-  container: linux-x64
```
Comment (anthonydresser) [inline build/azure-pipelines/sql-product-build.yml:11]: we need this container.

Comment (alanrenmsft) [reply build/azure-pipelines/sql-product-build.yml:11]: does this mean, we will use this container on our own agent?

**Comment by alanrenmsft** [comment]
the build is passing with docker enabled on the agent. but we are going with this solution instead: https://github.com/microsoft/azuredatastudio/pull/10775

**Pull request closed without merging**